### ISSUE TITLE
country-eforms_country.json Vatican City State codelist error?

### ISSUE THREAD

**Issue by Havunen** [OP-TED/eForms-SDK#815]
Updating eForms SDK Between 1.7.0 --> 1.10.1 causing weird translation for vatican city state: 

![image](https://github.com/OP-TED/eForms-SDK/assets/2021355/1baf017d-b3e9-4e95-a8be-3adb3f8d4ac4)

**Comment by bertrand-lorentz** [comment]
The texts for Vatican were indeed changed in SDK 1.10.0.
The new translation in FI looks correct in the file we publish (those files are in UTF-8):
https://github.com/OP-TED/eForms-SDK/blob/1.10.1/translations/code_fi.xml#L432

The character "ä" correspond to the code point 00E4 in hexadecimal.

Your screenshot shows a JSON file that is not part of the SDK. Is this file supposed to be in UTF-8 ?
I guess the content of the SDK was processed to produce this file. Maybe this processing replaced the character "ä" with what you see ?

**Comment by Havunen** [comment]
> The texts for Vatican were indeed changed in SDK 1.10.0.

Yeah no problem with encoding. I mean the translated text is completely off. It should be "Vatikaanivaltio" as it was in 1.7.0 and now it is "Pyhä istuin" which means "holy seat"
